packages/block-editor/src/layouts/flex.js

```diff
@@ -11,6 +11,7 @@ import {
 	arrowDown,
```

Comment:
what about flow.js? no fallback support there?

Reply:
I noticed that too when I went to work on this feature — the Flow layout currently doesn't use any fallback value at all, so I didn't inject one where there isn't one already (both in `layout.php` and `flow.js`). From looking over the code again, it appears that blockGap styles in the Flow layout are only ever output if the blockGap has been opted into, in which case there'll always be a value, so a fallback value will never be reached. E.g. in the following block:

https://github.com/WordPress/gutenberg/blob/6de16981c7fef3db7cc4a9d6cfcb0ad85c916c62/packages/block-editor/src/layouts/flow.js#L168

It was the Flex layout that currently has a `0.5em` hard-coded fallback, and where I think this feature is useful based on the current implementations.

But, please do let me know if I missed something here! 🙂